back-end/requirements.txt

```diff
@@ -11,4 +11,5 @@ sqlparse==0.5.0
 django-cors-headers==4.3.1
 drf-spectacular==0.27.2
 whitenoise==6.6.0
-gunicorn
\ No newline at end of file
+gunicorn
+celery=5.4.0
```

Comment:
use `celery[rabbitmq]=={version}` this will install all other necessary dependencies for celery to work with rabbitmq